worker/upgradeseries/worker_test.go

```diff
@@ -56,32 +56,6 @@ func (s *workerSuite) SetUpTest(c *gc.C) {
 	s.done = make(chan struct{})
 }
 
-// TestFullWorkflow uses the the expectation scenarios from each of the tests
-// below to compose a test of the whole upgrade-series scenario, from start
-// to finish.
-func (s *workerSuite) TestFullWorkflow(c *gc.C) {
```

Comment:
I am uncomfortable with deleting this. It is a statement of expected workflow through the standard life cycle of an upgrade and in addition to confirming, it *demonstrates* the expected overarching operation of the worker.

That said it isn't as important to keep as the deleted test below.